### DISCUSSION TITLE
上传图片前，怎么显示本地图片预览图？

### DISCUSSION THREAD

**Discussion by klower** [NG-ZORRO/ng-zorro-antd#5700] [Q&A]
## What problem does this feature solve?
我需要先显示选择的本地图片预览图，然后一次上传

## What does the proposed API look like?
怎么使用？

**Comment by liwei511** [comment]
手动上传： https://ng.ant.design/components/upload/zh#components-upload-demo-upload-manually
图片预览： https://ng.ant.design/components/upload/zh#components-upload-demo-picture-card
